addons/dark-www/addon.json

```diff
@@ -2445,62 +2445,62 @@
       }
     },
     {
-      "name": "Scratch 2.0 (2015)",
```

Comment:
I think this name is fine - this color scheme was used when 2.0 was the current version of Scratch.

Reply:
I wouldn't call a 8 year old color palette "default colors" tbh

Reply:
I wouldn't call anything except the purple one default.

Reply:
Well, blue was the default one month ago. Maybe we can call it "vanilla blue" or whatever